data/property-sort-orders/smacss.txt

```diff
@@ -121,41 +167,66 @@ color
 font
 font-family
 font-size
+font-size-adjust
+font-stretch
+font-effect
+font-smooth
 font-smoothing
+-moz-osx-font-smoothing
+-webkit-font-smoothing
```

Comment:
Well to be fair, i added them because those two properties are mandatory when doing a custom font and it's commonly used.
But i think i'll remove them, since it's not standard at the moment.